benchmark/buffers/buffer-creation.js

```diff
@@ -9,10 +8,9 @@ const bench = common.createBenchmark(main, {
     'fast-alloc-fill',
     'fast-allocUnsafe',
     'slow-allocUnsafe',
-    'slow',
-    'buffer()'],
-  len: [10, 1024, 2048, 4096, 8192],
-  n: [1024]
+  ],
+  len: [10, 1024, 4096, 8192],
+  n: [6e5]
```

Comment:
SlowBuffer mostly works identical to the regular buffer besides having a different performance profile for very small buffers due to the internal pre-allocated 8kb buffer.

Due to using `slow-allocUnsafe` the other unsafe call seems redundant (the mainly executed code should be identical). The old `buffer()` part is deprecated and should not matter anymore for any default case.

`n` was reduced to a reasonable iteration count.